server/routes/index.js

```diff
@@ -0,0 +1,23 @@
+const router = require('express').Router();
+const config = require('../config');
+const path = require('path');
+const fs = require('fs');
+const httpStatus = require('http-status-codes');
+
+router.use(config.resources.videos, require('./videos'));
+router.use(config.resources.healthCheck, require('./health_check'));
+
+router.get('/', (req, res) => {
```

Comment:
This router has functionality - it servers a html file.
In that case it should have a name and not just seat under `routes/index.js` - It's not really the main router